src/base/bittorrent/session.h

```diff
@@ -539,7 +539,7 @@ namespace BitTorrent
 
         bool addTorrent_impl(CreateTorrentParams params, const MagnetUri &magnetUri,
                              TorrentInfo torrentInfo = TorrentInfo(),
-                             const QByteArray &fastresumeData = {});
+                             QByteArray fastresumeData = {});
```

Comment:
this change is for libt 1.1 right? if so please add a comment that says we can turn it into `const &` when using libt >= 1.2, or just add an `#if`.

Reply:
>this change is for libt 1.1 right? 

Yes.
>if so please add a comment that says we can turn it into const & when using libt >= 1.2, or just add an #if.

Or maybe I'll just add another variable in the method and change its value to never touch this one?

Reply:
>Or maybe I'll just add another variable in the method and change its value to never touch this one?

Ok.